opal/mca/pmix/pmix2x/pmix/src/mca/gds/ds12/gds_dstore.c

```diff
@@ -2804,6 +2808,16 @@ static pmix_status_t dstore_assign_module(pmix_info_t *info, size_t ninfo,
         }
     }
 
+<<<<<<< c632784ca34c467055eadcb4efe84a25e5a3911b:opal/mca/pmix/pmix2x/pmix/src/dstore/pmix_esh.c
```

Comment:
What memory footprint problem? Is there any issue on that?
I think that  @karasevb have fixed (at least) one of the possible causes he discovered when keys was sent in the message from the server to the client even if the dstore was used.